templates/pbuilderrc.erb

```diff
@@ -28,7 +28,7 @@ DEBIAN_SUITES=("squeeze" $UNSTABLE_CODENAME $TESTING_CODENAME $STABLE_CODENAME $
 UBUNTU_SUITES=(<%= @ubuntu_suites %>)
 <% else %>
 # List of Ubuntu suites. Update these when needed.
-UBUNTU_SUITES=("lucid" "precise" "trusty" "utopic" "vivid" "wily" "xenial" "yakkety")
+UBUNTU_SUITES=("lucid" "precise" "trusty" "vivid" "wily" "xenial" "yakkety")
```

Comment:
or wily? I think we're still building wily for some projects, but that should be phased out, and this would certainly force that...